resources/lib/check_previous_episode.py

```
@@ -54,27 +51,26 @@ def run(args):
-            # Sleep/wait for abort for 10 seconds
-            if Store.kodi_event_monitor.waitForAbort(1):
-                # Abort was requested while waiting. We should exit
-                break
-
-    footprints(False)
+        # DEFAULT - RUN AS A SERVICE & WATCH PLAYBACK EVENTS
+        else:
+            Logger.info("Listening to onAvStarted for episode playback.")
+            kodi_event_monitor = KodiEventMonitor()
+            # noinspection PyUnusedLocal
+            kodi_player = KodiPlayer()
```

Ruff (0.13.1)

Local variable `kodi_player` is assigned to but never used

Remove assignment to unused variable `kodi_player`

(F841)